### PR TITLE
Fix issue #253

### LINKED ISSUE #253: Pressing enter doesn't submit forms in Safari
On /login, enter button doesn't submit forms. I suspect the same problem exists on all pages.

### PR DESCRIPTION
**Pull request by alicoding**
fixes #253

### CODE REVIEW

**Comment by alicoding** [comment]
@secretrobotron r?

File: templates/components/form/form.jsx [hunk 1]
```diff
@@ -224,11 +231,14 @@ var Form = React.createClass({
...
    * "owner" components call form.processFormData on us
    */
-  processFormData: function(e) {
-    e.preventDefault();
-    this.validate((error, data) => {
-      WebmakerActions.onFormValidation({err: error, user: !!error ? false : JSON.parse(JSON.stringify(this.state))});
-    });
+  processFormData: function(evt, clicked) {
+    evt.preventDefault();
+    let keyCode = evt.keyCode;
```
Comment (secretrobotron) [inline templates/components/form/form.jsx]: Don't think you need to cache this var if it's just used once below. Am I missing something you're doing with it here?

Comment (alicoding) [inline templates/components/form/form.jsx]: I'm only using this in the same scope which should be fine?

Comment (alicoding) [inline templates/components/form/form.jsx]: ignore my message here. I'm a bit blur LOL

**Pull request closed without merging**